src/main/java/edu/usf/sas/pal/muser/model/MusicAnalysisModel.java

```diff
@@ -1021,6 +1023,7 @@ public void setAudioData(AudioData audioData) {
             "record_id",
             "player_event_type",
             "ui_event_type",
+            "event_current_time_in_iso",
```

Comment:
```suggestion
            "event_current_time_in_iso_utc",
```

Reply:
More of a question for downstream tools - does adding another column here break the analysis tools that ingest the CSV? In other words, do the analysis tools depend on column order?

Reply:
I am not sure about this either since I did not understand the python analysis code that much. We can ask Kanchan about it. But if for some reason it does affect, do you want me to add this column as the last column in the meantime? Or keep it as it is?

Reply:
Let's keep it as-is, but we should give Kanchan a heads up.

Reply:
Alright, I'll do that and ask her.

Reply:
So Kanchan just suggested that it might break on the power bi part and we should preferably add it as the last column.

Reply:
Ok, that's fine.